src/configure/generatorprofilesha1values.in.h

```diff
@@ -0,0 +1,30 @@
+/*
+Copyright libCellML Contributors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#pragma once
+
+#include <string>
+
+namespace libcellml {
+
+/**
+ * This content of this file is generated, do not edit this file directly.
+ * See docs/dev_utilities.rst for further information.
+ */
+static const std::string C_PROFILE_SHA1 = "${C_PROFILE_SHA1_VALUE}";
+static const std::string PYTHON_PROFILE_SHA1 = "${PYTHON_PROFILE_SHA1_VALUE}";
```

Comment:
Rename to `C_GENERATOR_PROFILE_SHA1` and `PYTHON_GENERATOR_PROFILE_SHA1`, or `C_GEN_PROF_SHA1` and `PY_GEN_PROF_SHA1` for a shorter version. Same in `src/generator.cpp`.

Reply:
[e6a1541](https://github.com/cellml/libcellml/pull/1054/commits/e6a1541914edc61403663ced7d23188e6dbb5a16)